sympy/polys/factortools.py

```diff
@@ -124,13 +124,29 @@ def dmp_trial_division(f, factors, u, K):
 
 
 def dup_zz_mignotte_bound(f, K):
-    """Mignotte bound for univariate polynomials in `K[x]`. """
-    a = dup_max_norm(f, K)
-    b = abs(dup_LC(f, K))
-    n = dup_degree(f)
-
-    return K.sqrt(K(n + 1))*2**n*a*b
+    """
+    The Knuth-Cohen variant of Mignotte bound for
+    univariate polynomials in `K[x]`.
 
+    Reference: John Abbott, Journal of Symbolic Computation 50 (2013) 532–563.
```

Comment:
Please see other methods on how to include references.

Reply:
Can you please give me a link with this?

Reply:
Have a look at [references](https://github.com/sympy/sympy/blob/8adfdbed18e6a0e06d862213752d2cb96a84218a/sympy/polys/factortools.py#L182)

Reply:
Think i got it right!